backend/django/api/views.py

```
@@ -1,92 +1,217 @@
-    queryset = Project.objects.prefetch_related("tasks").order_by("-created_at")
+    queryset = Project.objects.prefetch_related(
+        Prefetch(
+            "tasks",
+            queryset=Task.objects.only(
+                "id", "title", "done", "project_id", "created_at"
+            ),
+        )
+    ).order_by("-created_at")
+
     serializer_class = ProjectSerializer
     permission_classes = [IsAuthenticated]
```

Ruff (0.14.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)